nss-day-cohort-41/tabloid-fullstack-loch-ness-cryptozoologists
# Jrr

Issue #19 View A Post's Comments:
Upon viewing a post the user can select details. In the details page the user can click on the 'Comments' link. This will then take the user to a list view of all of the comments associated with only that post. 

Changes made:
Changes made to Application Views, Header.js, **Comment link added to PostDetails.cs**
 Model, repository, and controller added for comments.
Provider and list added for comments on client side. 

Testing Instructions:
Git fetch --all
git checkout 'my branch'
make sure to run both your server and client side apps with data in your API
view a post then view a post's details then click on comments.
